js/models/addressBook_model.js

```diff
@@ -2,12 +2,17 @@ angular.module('contactsApp')
 .factory('AddressBook', function()
 {
 	return function AddressBook(data) {
+		var storageKey = btoa(data.data.href.split('/').splice(-3, 2).join('_'));
```

Comment:
This variable isn't used later on. Also stuff like this asks for a short comment.

Reply:
Fixed